front/app/api/idea_statuses/useIdeaStatuses.ts

```diff
@@ -7,10 +7,11 @@ import { IIdeaStatuses, IdeaStatusesKeys } from './types';
 const fetchIdeaStatuses = () =>
   fetcher<IIdeaStatuses>({ path: '/idea_statuses', action: 'get' });
 
-const useIdeaStatuses = () => {
+const useIdeaStatuses = (enabled = true) => {
```

Comment:
Usually we make this an object so that the parameters are named. Not a big deal here but it would make it more explicit.